app/src/main/java/org/fossasia/openevent/general/MainActivity.kt

```diff
@@ -21,8 +21,10 @@ import org.koin.androidx.viewmodel.ext.android.viewModel
 const val PLAY_STORE_BUILD_FLAVOR = "playStore"
 const val EVENT_IDENTIFIER = "eventIdentifier"
 const val VERIFICATION_TOKEN = "verificationToken"
+const val RESET_PASSWORD_TOKEN = "resetPasswordToken"
 private const val VERIFY = "verify"
 private const val TOKEN = "token"
+private const val RESET_PASSWORD = "reset-password"
```

Comment:
camel case

Reply:
It is according to the url: Please use the following link to reset your password.
https://eventyay.com/reset-password?token=-----------